### ISSUE TITLE
Separate highlighter to represent the actual changes in the diff

### ISSUE THREAD

**Issue by dhruvinsh** [NeogitOrg/neogit#374]
Thank you for this awesome plugin. I was wondering if it is possible to have some addition highlight to show the actual changes with bright color and hence have separate highlighter if possible. ( I have no clue whether its even possible). Below is the example shows the change I am requesting for

**From Neogit:**
![image](https://user-images.githubusercontent.com/33428164/196450700-655e4927-d523-4001-810a-fe8220ce722d.png)

**From Magit (doomemacs)**
![image](https://user-images.githubusercontent.com/33428164/196450896-c428cb41-e14a-47ed-a9f9-a8758ad0d73e.png)

**Comment by ten3roberts** [comment]
It is most certainly possible. https://github.com/sindrets/diffview.nvim does it, and the neogit `d` diffview integration also works with the exact highlighting.

![image](https://user-images.githubusercontent.com/25723553/198904146-5cb17186-6fd7-4fc3-8253-18f02df0bd18.png)

**Comment by dhruvinsh** [comment]
Yes correct, with diffview it is visible, not in the same color palette but yes its there. It would be great to see the similar highlights with neogit native diff shown per file via <tab> key.
